doc/nrf/ug_nrf91_features.rst

```diff
@@ -160,6 +160,175 @@ The :ref:`nrfxlib:nrf_modem` is released as an OS-independent binary library in
+This results in solving an equation system with four unknowns, and therefore a minimum of four satellites must be tracked to acquire a fix.
+
+Enhancements to GNSS
+====================
+
+When GNSS has not been in use for a while or if the device is in relatively weak signaling conditions, it might take longer to acquire a fix.
+To improve this, Nordic Semiconductor has implemented the following methods for acquiring a fix in a shorter time:
+
+•	A-GPS or P-GPS or a combination of both
+•	Low accuracy mode
+
+Assisted GPS (A-GPS)
+---------------------
+
+A-GPS is commonly used to improve the Time to first fix (TTFF) by utilizing a connection (for example, over cellular) to the internet to retrieve the Almanac and Ephemeris data.
+A connection to an internet server that has the Almanac and Ephemeris data is several times quicker than using the slow 50 bps data link to the GPS satellites.
+There are many options to retrieve this A-GPS data.
+Two such options are using `nRF Cloud`_ and SUPL.
+|NCS| provides example implementations for both these options.
+The A-GPS solution available through nRF Cloud has been optimized for embedded devices to reduce protocol overhead and data usage.
+This in turn results in the download of reduced amount of data, thereby reducing time, power consumption, and data costs.
+See the section on A-GPS data in :ref:`nrfxlib:gnss_interface` for more information about the retrieval of A-GPS data.
+
+Predicted GPS (P-GPS)
+---------------------
+
+P-GPS is a form of assistance, where the device can download up to two weeks of predicted satellite Ephemerides data.
+Normally, devices connect to the cellular network approximately every two hours for up-to-date satellite Ephemeris information or they download the Ephemeris data from the acquired satellites.
+P-GPS enables devices to determine the exact orbital location of the satellite without connecting to the network every two hours with a trade-off of reduced accuracy of the calculated position over time.
+Note that P-GPS requires more memory compared to regular A-GPS.
+
+Also, note that due to satellite clock inaccuracies, not all functional satellites will have Ephemerides data valid for two weeks in the downloaded PGPS package.
+This means that the number of satellites having valid predicted Ephemerides reduces in number roughly after ten days.
+Hence, the GNSS module needs to download the Ephemeris data from the satellite broadcast if no predicted Ephemeris is found for that satellite to be able to use the satellite.
+
+SUPL compared to nRF Cloud as location service
+----------------------------------------------
+
+SUPL library uses approximately 100 kB of memory with a bit more overhead, which costs data and results in power consumption because you must have the LTE radio on for a longer period. nRF Cloud as location services instead has been designed to use minimal memory space on the device and send minimal possible data over the air to save power.
```

Comment:
memory = flash memory, I assume. It would be good to add a small note on flash and RAM usage.

Reply:
And I don't think we can say that SUPL has more overhead. The protocol has been optimized for cellular networks, so it has very little overhead. When getting all assistance data, SUPL is more efficient than nRF Cloud over MQTT. I'm assuming that the same is also true when getting only ephemerides. The difference is small, though.